core/src/com/unciv/logic/map/tile/Tile.kt

```diff
@@ -56,13 +61,27 @@ class Tile : IsPartOfGameInfoSerialization {
     var resourceAmount: Int = 0
 
     var improvement: String? = null
-    var improvementInProgress: String? = null
```

Comment:
Players will lose all existing improvements in progress. This is not a game-changing (literally) loss, but it is something we want to avoid - especially since in MP games if one is ahead and one is behind, they'll *both* lose their ability to construct things! BAD

Something to add to BackwardsCompatibility.

Reply:
You tested this?